gcloud-java-examples/src/main/java/com/google/gcloud/examples/dns/snippets/ManipulateZonesAndRecordSets.java

```diff
@@ -23,6 +23,7 @@
 package com.google.gcloud.examples.dns.snippets;
 
 import com.google.gcloud.dns.ChangeRequest;
+import com.google.gcloud.dns.ChangeRequestInfo;
```

Comment:
`isDone()` and `reload()` will come in a separate PR. I made a note in #679.

`zone.listChangeRequests().iterateAll()` on line 112 returns `Iterator<ChangeRequest>`